### PR TITLE
#808 - Fixed this by showing a snack bar message if the chosen PDF do…

### PR DESCRIPTION
**Pull request by svignesh93**
…c has no text

## Description:

Fixed this by showing a snack bar message if the chosen PDF doc has no text

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.2'
+        classpath 'com.android.tools.build:gradle:3.5.1'
```
Comment (Swati4star) [inline build.gradle]: revert this change

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Mon Aug 12 00:16:23 PDT 2019
+#Fri Oct 11 15:38:07 IST 2019
```
Comment (Swati4star) [inline gradle/wrapper/gradle-wrapper.properties]: revert this change.
This is causing merge conflicts too

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -145,6 +145,7 @@
     <string name="snackbar_imagecropped">Images cropped</string>
     <string name="cropImage_activityTitle">Crop Image</string>
     <string name="snackbar_pdfCreated">PDF created!</string>
+    <string name="snack_bar_empty_txt_in_pdf">The chosen PDF has no text to extract!</string>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml]: can you add this string to other locale files also (in english only)? - just an addition

File: build.gradle (hunk 1 above)
Comment (svignesh93) [reply build.gradle]: The master branch has the same version 3.5.1, Please check this

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (svignesh93) [reply app/src/main/res/values/strings.xml]: Yes, I've added localisation for the new string

**Comment by Swati4star** [comment]
Great @svignesh93 

Congrats on your first contribution to this project! 💯
